src/signals/incident/components/form/MapSelectors/Asset/types.ts

```diff
@@ -64,3 +65,13 @@ export interface AssetSelectValue {
   setMessage: (message?: string) => void
   update: (items: Item[]) => void
 }
+
+export interface DataLayerProps {
```

Comment:
Shouldn't this be defined in the component file itself?

Reply:
sure